config/subsidy-application-management/forms/climate/oproep-2/step-submit-opvolgmoment-2024/versions/20231116154454/form.json

```diff
@@ -35,6 +35,17 @@
             ]
           }
         ]
+      },
+      {
+        "s-prefix": "lblodSubsidie:attachmentLEKP",
```

Comment:
I thought I had to use another predicate, to not overwrite/mess with the previous LEKP attachment

Reply:
Yes, you're right; I missed that part. LEKP 1.0 is one of the older subsidies I think, which is why the document field had a plain `lblodSubsidie:attachment` prefix whereas other subsidies are more clear when it comes to document prefix names. I was thinking of adding "report" to the name (`attachmentLEKPReport`) to mimic the field's description inside the form, but that's a bit of a subjective opinion. We can wait for Achraf's input to see what choice is best.

Reply:
@aatauil which naming should we use for this new attachment?

Reply:
Not liking `lblodSubsidie:attachment` but it seems to be used for all climate subsidies so leaving it like that for consistency. 

For the second upload field, going with attachmentLEKPReport because its a bit clearer.